source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveInputValue.cs

```diff
@@ -363,7 +363,7 @@ public override string GetValue()
                         AdaptiveChoice adaptiveChoice = item.DataContext as AdaptiveChoice;
                         return adaptiveChoice.Value;
                     }
-                    return null;
+                    return "";
                 }
```

Comment:
Can you add a quick code comment here explaining what this will do- are we stripping out original empty strings before teh cde gets here that needs us to manually return an empty string?